### ISSUE TITLE
Notice about: Session: Creating a chrome extension

### ISSUE THREAD

**Issue by saidbakr** [medhatdawoud/AMA#42]
**Session**: Creating a chrome extension
**type**: series
**period range**: 1h to 1.5h in total
**expected**: Understand how to create a chrome extension using JavaScript and build a real one together, maybe re-creating [RTL Fixer](https://chrome.google.com/webstore/detail/mkeclhaiiffmeeokkomoameekkpiiecf/publish-accepted?authuser=0&hl=en) extension

originally suggested in #21 by @MAlngaawy

_Originally posted by @medhatdawoud in https://github.com/medhatdawoud/AMA/issues/28#issuecomment-609082812_

> **Session**: Creating a chrome extension
> **type**: series
> **period range**: 1h to 1.5h in total
> **expected**: Understand how to create a chrome extension using JavaScript and build a real one together, maybe re-creating [RTL Fixer](https://chrome.google.com/webstore/detail/mkeclhaiiffmeeokkomoameekkpiiecf/publish-accepted?authuser=0&hl=en) extension
> 
> originally suggested in #21 by @MAlngaawy

**For this video, I'd like to mention "why should we void the easy way of implementation for that extension?" by injecting CSS:**
```
body * {
direction: rtl !important;
}
code {
direction: ltr !important;
}
```

**Comment by medhatdawoud** [comment]
Don't understand what you meant.

you could have added this comment on your closed ticket, why opening a new issue, such a spammy way of communication that I don't like 👎

**Comment by saidbakr** [comment]
![image](https://user-images.githubusercontent.com/467554/78461308-1967ef00-76c8-11ea-8907-427ec3d5b8bc.png)

**Comment by medhatdawoud** [comment]
That's a nice catch, I'll put it in mind during the new series isA

**Comment by saidbakr** [comment]
> Don't understand what you meant.
> 
> you could have added this comment on your closed ticket, why opening a new issue, such a spammy way of communication that I don't like -1

In the closed issue, you had mentioned to the issue #28. Since, closed means closed, so it is not properly to reopen it while the GitHub allows an option to **Reference in new issue** for your comment there , additionally, the closed issue has another topic. It does not talk about a video request.
Best regards,
Said Bakr.
